x-pack/platform/plugins/shared/streams/server/routes/streams/management/route.ts

```diff
@@ -102,8 +102,32 @@ export const getStreamsStatusRoute = createServerRoute({
   },
 });
 
+export const getClassicStreamsStatusRoute = createServerRoute({
+  endpoint: 'GET /api/streams/_classic_status',
+  options: {
+    access: 'internal',
+  },
+  security: {
+    authz: {
+      requiredPrivileges: [STREAMS_API_PRIVILEGES.read],
+    },
+  },
+  handler: async ({ request, getScopedClients }): Promise<{ can_manage: boolean }> => {
+    const { scopedClusterClient } = await getScopedClients({ request });
+
+    const REQUIRED_MANAGE_PRIVILEGES = ['manage_index_templates'];
```

Comment:
do we also need `create_index` or `manage_pipeline` permissions to create the data stream ?